app/api/v1/router.py

```diff
@@ -15,6 +15,12 @@
 from app.api.v1.endpoints.season_start_probabilities.router import (
     router as season_start_probabilities_router,
 )
+from app.api.v1.endpoints.seasonal_forecast_list.router import (
+    router as seasonal_forecast_list_router,
+)
+from app.api.v1.endpoints.seasonal_forecast_pdf.router import (
```

Comment:
nit(non-blocking)
Given that retrieving a pdf depends on passing a district name as identified from the list endpoint, I might be tempted to suggest restructuring all under a single endpoint, e.g.

`/seasonal_forecast` - lists available seasonal forecasts
`/seasonal_forecast/{country_id}` - return pdf for country (not sure if they're on webpage but should be in use)
`/seasonal_forecast/{country_id}/{district_id}` - return the pdf for downscaled district forecast